composer.json

```
@@ -10,7 +10,9 @@
     ],
     "license": "BSD-3-Clause",
     "require": {
-        "silverstripe/framework": "^4.9"
+        "silverstripe/framework": "^4.9",
+        "ext-openssl": "*",
+        "silverstripe/cms": "^4.11"
```

So, I *think* that if you add the php version here, then the github actions tests will actually respect it.

```suggestion
        "silverstripe/cms": "^4.11",
        "php": "^8.0 || ^8.1"
```

If they don't then there is a parameter `dynamic_matrix: true` which can be added...which, uh, might work 😎

---

Awesome, the addition of setting the php dependency works out of the box. All green ticks now 👍